requirements/testing.txt

```diff
@@ -1,2 +1,3 @@
 # Tox will setup its own environment and so it doesn't depend on anything
 tox<4.0
+beautifulsoup4==4.11.1
```

Comment:
Why is this needed? Shouldn't tox install everything it needs?

Reply:
It's currently in the analyzer requirements. Probably makes sense to just move it to the base requirements?

Reply:
I get it. We need this for the naive analyzer (for tests) but because we aren't installing analyzer requirements this doesn't get installed. Maybe move this one to base but I do understand the desire to have requirements per-analyzer.

Reply:
Yea.. I'll leave it here for now, but will likely end up in some kind of analyzer base requirements or similar.